lib/media/drm_engine.js

```diff
@@ -727,11 +738,19 @@ shaka.media.DrmEngine = class {
           // MediaKeySystemConfiguration
           config['drmInfos'].push(info);
 
+          if (info.initData && info.initData.length) {
+            config.initDataTypes = [
+              ...new Set(info.initData.map(({initDataType}) => initDataType)),
```

Comment:
This is a bit dense for an uninformed reader.  I suggest removing the object unpacking and just using a normal argument instead.